tests/c/varargs.c

```diff
@@ -53,7 +53,7 @@ int main(int argc, char **argv) {
   NOOPT_VAL(i);
   while (i > 0) {
     yk_mt_control_point(mt, &loc);
-    int res = varargfunc(3, argc*1, 2, 3);
+    int res = varargfunc(3, argc * 1, 2, 3);
```

Comment:
`argc * 1` -> `argc`?

Reply:
Hah.

Reply:
Fixed in [ba877c6](https://github.com/ykjit/yk/pull/568/commits/ba877c6d94bbc299f3dd8d71bc86b37952245c6a)